test/integration/egress.go

```diff
@@ -31,6 +34,13 @@ func (t *egress) String() string {
 }
 
 func (t *egress) setup() error {
+	if !t.Egress {
+		return nil
+	}
+	if err := util.Run(fmt.Sprintf(
+		"kubectl -n %s apply -f test/integration/testdata/external-service.yaml", t.Namespace)); err != nil {
+		return err
+	}
```

Comment:
I'd like to keep it as is since it follows the same pattern for multi-step setups. it will be easier to add another step.